terragrunt/aws/sso_proxy/configs/routes.yml.tmpl

```diff
@@ -47,9 +47,13 @@
   cors_allow_preflight: true
   policy:
       - allow:
-          and:
+          or:
             - http_method:
                 is: POST
+            - http_path:
+                is: /api/v1/bom
+          or:
+            - http_method:
```

Comment:
is the `AND` implied when two items are listed in the `or:` ? wondering if its a or. If this doesn't work we can try the below

```
      - allow:
          or:
            - http_method:
                is: POST
            - http_method:
                is: PUT
          and:
            - http_path:
                is: /api/v1/bom
```